src/vaults/BalanceSheet.sol

```diff
@@ -178,42 +214,27 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
     // View methods
     //----------------------------------------------------------------------------------------------
 
+    /// @inheritdoc IBalanceSheet
+    function escrow(PoolId poolId) public view returns (IPoolEscrow) {
```

Comment:
This seems useful for balance sheet managers, to know e.g. where to transfer after noteDeposit